roles/cluster_ensure_machineset/tasks/derive_machineset.yml

```diff
@@ -6,9 +6,22 @@
   register: oc_get_machinesets
```

Comment:
```
roles/cluster_ensure_machineset/tasks/main.yml:15
TASK: cluster_ensure_machineset : Create the new machineset
  - include:	derive_machineset.yml
included: /home/kevin/openshift/ci-artifacts/roles/cluster_ensure_machineset/tasks/derive_machineset.yml for localhost
```
could you change the `name` there to: `Derive the new machineset`? the `Create` is a bit confusing here, sounds like a simple/single task